src/kotlin/kotlin.test.ts

```diff
@@ -0,0 +1,647 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { testTokenization } from '../test/testRunner';
+
+testTokenization('kotlin', [
```

Comment:
I'll be adding a couple more tests here that I wrote for the following lines: 

```
inline fun <reified T : Any> foo()',

'val x: X=5'
```

Reply:
👍